sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,586 @@
+# Two-Clients Approach
+
+## APIs:
+class SeviceBusSenderClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        entity_name : str,
+        credential : TokenCredential,
+        logging_enable: bool = False,
```

Comment:
Are these kwargs after "credential"?

Reply:
yes, credential is a `must` parameter in the constructor.